Source/Core/ClippingPlaneCollection.js

```diff
@@ -362,6 +364,15 @@ define([
         return intersection;
     };
 
+    /**
+     * Determines if rendering with clipping planes is supported.
+     *
+     * @returns {Boolean} <code>true</code> if ClippingPlaneCollections are supported; otherwise, returns <code>false</code>
+     */
+    ClippingPlaneCollection.isSupported = function() {
```

Comment:
Does this warrant an update to CHANGES.md, or too minor? @pjcozzi?

Reply:
Anything that changes the public API always warrants a CHANGEs.md update.